packages/thumbnail/README.md

```diff
@@ -26,72 +26,148 @@ import { Thumbnail } from '@spectrum-web-components/thumbnail';
 
 ## Sizes
 
-<sp-tabs selected="m" auto label="Size Attribute Options">
-<sp-tab value="xss">Extra Extra Small</sp-tab>
-<sp-tab-panel value="xxs">
+<sp-tabs selected="500" auto label="Size Attribute Options">
+<sp-tab value="50">50</sp-tab>
+<sp-tab-panel value="50">
 
 ```html
-<sp-thumbnail size="xxs">
+<sp-thumbnail size="50" style="display: block;">
```

Comment:
We be adding this inline styling to the class for thumbnail in the spectrum-css PR and do another beta release. So all these will `style="display: block;`  be removed before the PR is ready to go.

Reply:
these have been removed!